src/measures/MultiS3.cpp

```diff
@@ -32,17 +56,31 @@ void MultiS3::initDegrees(const Graph& G2)
         for (uint j = 0; j < G2AdjLists[i].size(); ++j)
         {
             neighbor = G2AdjLists[i][j];
-            shadowDegrees[i] += G2Matrix[i][neighbor]; // +1 if G1 has edge here, since it was pruned?
+            totalDegrees[i] += G2Matrix[i][neighbor];

```

Comment:
Here is where we put the +1, rather than changing G2 above.

Reply:
G2 is a copy above so it's OK.
That copy's gonna be made regardless because of the original author's code for getAdjMatrix

Reply:
Yup.